app/src/main-process/main.ts

```diff
@@ -104,6 +105,15 @@ if (__WIN32__ && process.argv.length > 1) {
   }
 }
 
+if (__DARWIN__) {
```

Comment:
We're doing this platform check here and in `shellNeedsPatching`. Let's pick one place to do it.